theano/gpuarray/dnn.py

```diff
@@ -1393,6 +1393,9 @@ def local_abstractconv_cudnn(node):
     inp1 = node.inputs[0]
     inp2 = node.inputs[1]
 
+    if (node.op.filter_dilation != (1, 1)):
```

Comment:
Could we do here as Lasagne do and use of the grad conv from cudnn in some condition? We should time it before. Just in case someone is interrested to spend time on this.